### DISCUSSION TITLE
Docker pack v0.23.0 raises `ERROR: failed to launch: bash exec: no such file or directory`

### DISCUSSION THREAD

**Discussion by DamnOpsGG** [buildpacks/community#158] [General]
Hey there,

I've tried the latest version of docker pack, it raises the following error even though I ran docker run command with help.
```
docker run buildpacksio/pack:0.23.0 help
ERROR: failed to launch: bash exec: no such file or directory
```
Any idea?

**Comment by jromero** [comment]
That's really interesting. I did verify the issue and that it was not an issue in the previous version image `buildpacksio/pack:0.22.0`.

```
❯ docker run buildpacksio/pack:0.22.0 version
0.22.0

❯ docker run buildpacksio/pack:0.23.0 version
ERROR: failed to launch: bash exec: no such file or directory
```

@dfreilich, do you have any ideas?

**Reply by dfreilich** [reply]
Hmmm. No, but the same issue occurs with `v0.24.0`
```
> docker run buildpacksio/pack:0.24.0 version
ERROR: failed to launch: bash exec: no such file or directory
```

Definitely worth looking into. Thanks for pointing that out, @marsladder-github!

**Comment by dfreilich** [comment]
Inspecting the image makes it clear what's happening: 

### `> pack inspect buildpacksio/pack:0.22.0`
```
Inspecting image: buildpacksio/pack:0.22.0
...
Processes:
  TYPE                 SHELL        COMMAND                                                    ARGS
  web (default)                     /layers/paketo-buildpacks_go-build/targets/bin/pack
  pack                              /layers/paketo-buildpacks_go-build/targets/bin/pack
```
vs
### `> pack inspect buildpacksio/pack:0.23.0`
```
Inspecting image: buildpacksio/pack:0.23.0
...
Processes:
  TYPE        SHELL        COMMAND        ARGS
  pack                     /layers/paketo-buildpacks_go-build/targets/bin/pack
```

No default process is being set in the image, after 0.23.0

**Reply by dfreilich** [reply]
Sorry it took me a while to get around to it, @marsladder-github , but both images should be fixed at this point! (the PR for it are here: https://github.com/buildpacks/pack/pull/1382)
